docs/README.md

```diff
@@ -22,6 +22,20 @@ algob init my-project
 ```
 This will create a directory `my-project` and put multiple files into it.
 
+In the `my-project` folder you'll have following items:
+
+* `assets/`: Directory for assets and contracts files
+
+* `scripts/`: Directory for scripts to deploy and run your assets and contracts
+
+* `tests/`: Directory for test files for testing your assets and contracts
+
+* `algob.config.js`: Algob configuration file
```

Comment:
please remove the empty new lines in the list above.

Reply:
ok